internal/metrics/metrics.go

```diff
@@ -132,15 +132,10 @@ func (m Metrics) MirrorLastUpdatedGauge() prometheus.Gauge {
 	return m.mirrorLastUpdatedGauge
 }
 
-func UpdateAndPushEndJobMetrics(m *Metrics, startTime time.Time, cfg *config.Config, reg *prometheus.Registry) {
-	crawlerDuration(m, startTime)
+func UpdateAndPushEndJobMetrics(m *Metrics, startTime time.Time, cfg *config.Config) {
```

Comment:
This function doesn't push the metrics, but the name suggests it does. Which is right?